data/rtl-config-vhdl-spec.json

```
@@ -0,0 +1,292 @@
+[
+  {
+    "name": "handshake.addf",
+    "generator": "python $DYNAMATIC/experimental/tools/unit-generators/vhdl/vhdl-unit-generator.py -n $MODULE_NAME -t addf -p port_types=$PORT_TYPES >> $OUTPUT_DIR/$MODULE_NAME.vhd"
+  },
```

```suggestion
  {
    "name": "handshake.addf",
    "generator": "python3 $DYNAMATIC/experimental/tools/unit-generators/vhdl/vhdl-unit-generator.py -n $MODULE_NAME -t addf -o $OUTPUT_DIR/$MODULE_NAME.vhd -p port_types=$PORT_TYPES"
  },
```
This was a suggestion from @Jiahui17 that I applied also in https://github.com/EPFL-LAP/dynamatic/pull/270.
The idea is that we shouldn't rely on bash functionalities like `>>` for the generator.